include/spirv-tools/optimizer.hpp

```diff
@@ -838,6 +838,14 @@ Optimizer::PassToken CreateWrapOpKillPass();
 // capabilities.
 Optimizer::PassToken CreateAmdExtToKhrPass();
 
+// Replaces the internal version of GLSLstd450 OpInterpolateAt* instructions
+// with the externally valid version. The internal version allows an OpLoad
+// of the interpolant for the first argument. This pass removes the OpLoad and
+// replace with itspointer. glslang creates the internal version for HLSL. This
```

Comment:
The tense for "replace" should be the same as "removes".  So either "This pass will remove the OpLoad and replace it with its pointer", or "This pass removes the OpLoad and replaces it with its pointer."

Reply:
OK.